Google.Api.Generator.Tests/ProtoTests/Lro/LroClient.cs

```diff
@@ -0,0 +1,135 @@
+﻿using gax = Google.Api.Gax;
+using gaxgrpc = Google.Api.Gax.Grpc;
+using lro = Google.LongRunning;
+using proto = Google.Protobuf;
+using grpccore = Grpc.Core;
+using grpcinter = Grpc.Core.Interceptors;
+using sys = System;
+using scg = System.Collections.Generic;
+using sco = System.Collections.ObjectModel;
+using st = System.Threading;
+using stt = System.Threading.Tasks;
+
+namespace Testing.Lro
+{
+    public abstract class LroClient
```

Comment:
No LroClientImpl?

Reply:
These tests, and some of the other tests are badly named.
They are not testing the core LRO functionality, that's tested in `BasicLro`. This test is for advanced LRO functionality - in this case it's only testing that the method-signature and resource-name parts work properly.
So all the "normal" LRO code is deliberately missing from this test.

I'll refactor in a future PR to re-name the tests to be more descriptive.